downstream/modules/platform/proc-create-keycloak-client.adoc

```diff
@@ -0,0 +1,143 @@
+[id="proc-create-keycloak-client_{context}"]
+
+= Creating a client
+
+Use the following procedure to create the Client.
+
+.Procedure
```

Comment:
It's really just a preference but I'd add an empty line between the title and first step in the procedure.